Resolve conflicts in mark-testing-branch before merge to master

JoelProminic opened the pull request:
Thinking about the #146 merge further, I think we can resolve some of these conflicts in advance by merging master into mark-testing-branch.  I created this pull request for further discussion and review.

We can discuss this further in the meeting tomorrow.

MarkProminic reviewed (approved):
I think that if this doesn't break my renames of the DemoTasks.hx and DemoTasksHostFileGenerator.hx, this is fine

JoelProminic commented:
Unfortunately, I see more conflicts after this merge:

```
Super.Human.Installer % git checkout intermediary-branch-to-marks-branch
Switched to branch 'intermediary-branch-to-marks-branch'
Your branch is up to date with 'origin/intermediary-branch-to-marks-branch'.
Super.Human.Installer % git merge --no-commit --no-ff mark-testing-branch
CONFLICT (rename/rename): Assets/provisioners/additional-prov/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml renamed to Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml in HEAD and to Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml in mark-testing-branch.
CONFLICT (rename/rename): Assets/provisioners/additional-prov/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml renamed to Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml in HEAD and to Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml in mark-testing-branch.
CONFLICT (rename/rename): Assets/provisioners/additional-prov/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml renamed to Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml in HEAD and to Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml in mark-testing-branch.
CONFLICT (rename/rename): Assets/provisioners/additional-prov/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2 renamed to Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2 in HEAD and to Assets/provisioners/hcl_domino_standalone_provisioner/0.1.20/scripts/ansible/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2 in mark-testing-branch.
CONFLICT (rename/rename): Assets/provisioners/demo-tasks/0.1.20/scripts/ansible/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2 renamed to Assets/provisioners/hcl_domino_standalone_provisioner/0.1.20/scripts/ansible/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2 in HEAD and to Assets/provisioners/hcl_domino_standalone_provisioner/0.1.22/scripts/provisioners/ansible/roles/hcl_domino_traveler_htmo/templates/htmo-traveler-access.json.j2 in mark-testing-branch.
CONFLICT (rename/rename): Assets/provisioners/demo-tasks/0.1.22/scripts/provisioners/ansible/roles/hcl_domino_traveler_htmo/templates/htmo-traveler-access.json.j2 renamed to Assets/provisioners/hcl_domino_standalone_provisioner/0.1.22/scripts/provisioners/ansible/roles/hcl_domino_traveler_htmo/templates/htmo-traveler-access.json.j2 in HEAD and to Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2 in mark-testing-branch.
CONFLICT (rename/rename): Assets/provisioners/demo-tasks/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml renamed to Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml in HEAD and to Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml in mark-testing-branch.
CONFLICT (rename/rename): Assets/provisioners/demo-tasks/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml renamed to Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml in HEAD and to Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml in mark-testing-branch.
CONFLICT (rename/rename): Assets/provisioners/demo-tasks/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml renamed to Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml in HEAD and to Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml in mark-testing-branch.
CONFLICT (rename/rename): Assets/provisioners/demo-tasks/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2 renamed to Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2 in HEAD and to Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2 in mark-testing-branch.
Auto-merging Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml
CONFLICT (add/add): Merge conflict in Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml
Auto-merging Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml
CONFLICT (add/add): Merge conflict in Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml
Auto-merging Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml
CONFLICT (add/add): Merge conflict in Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml
Auto-merging Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2
CONFLICT (add/add): Merge conflict in Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2
Auto-merging Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml
CONFLICT (add/add): Merge conflict in Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml
Auto-merging Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml
CONFLICT (add/add): Merge conflict in Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml
Auto-merging Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml
CONFLICT (add/add): Merge conflict in Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml
Auto-merging Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2
CONFLICT (add/add): Merge conflict in Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2
Auto-merging Source/SuperHumanInstaller.hx
CONFLICT (content): Merge conflict in Source/SuperHumanInstaller.hx
Auto-merging Source/superhuman/managers/ProvisionerManager.hx
CONFLICT (content): Merge conflict in Source/superhuman/managers/ProvisionerManager.hx
Auto-merging Source/superhuman/server/AdditionalServer.hx
Auto-merging Source/superhuman/server/Server.hx
CONFLICT (content): Merge conflict in Source/superhuman/server/Server.hx
Auto-merging Source/superhuman/server/provisioners/ProvisionerType.hx
CONFLICT (content): Merge conflict in Source/superhuman/server/provisioners/ProvisionerType.hx
Auto-merging project.xml
CONFLICT (content): Merge conflict in project.xml
Automatic merge failed; fix conflicts and then commit the result.
Super.Human.Installer % git status
On branch intermediary-branch-to-marks-branch
Your branch is up to date with 'origin/intermediary-branch-to-marks-branch'.

You have unmerged paths.
  (fix conflicts and run "git commit")
  (use "git merge --abort" to abort the merge)

Changes to be committed:
	modified:   .github/workflows/man-release.yml
	modified:   .github/workflows/sub-build-windows.yml
	new file:   Assets/provisioners/hcl_domino_additional_provisioner/provisioner.yml
	new file:   Assets/provisioners/hcl_domino_standalone_provisioner/provisioner.yml
	new file:   Assets/provisioners/provisioner.yml.sample
	modified:   Assets/text/en_US.json
	modified:   Source/superhuman/components/AdvancedConfigPage.hx
	modified:   Source/superhuman/components/ConfigPage.hx
	new file:   Source/superhuman/components/DynamicAdvancedConfigPage.hx
	new file:   Source/superhuman/components/DynamicConfigPage.hx
	modified:   Source/superhuman/components/RolePage.hx
	modified:   Source/superhuman/components/ServerList.hx
	modified:   Source/superhuman/components/ServerPage.hx
	modified:   Source/superhuman/components/SettingsPage.hx
	modified:   Source/superhuman/components/additionals/AdditionalServerPage.hx
	modified:   Source/superhuman/components/serviceType/ServiceTypePage.hx
	modified:   Source/superhuman/config/SuperHumanGlobals.hx
	modified:   Source/superhuman/config/SuperHumanHashes.hx
	modified:   Source/superhuman/events/SuperHumanApplicationEvent.hx
	modified:   Source/superhuman/managers/ServerManager.hx
	modified:   Source/superhuman/server/AdditionalServer.hx
	modified:   Source/superhuman/server/data/ServerData.hx
	modified:   Source/superhuman/server/data/ServiceTypeData.hx
	modified:   Source/superhuman/server/definitions/ProvisionerDefinition.hx
	modified:   Source/superhuman/server/hostsFileGenerator/AdditionalProvisionerHostsFileGenerator.hx
	renamed:    Source/superhuman/server/hostsFileGenerator/DemoTasksHostsFileGenerator.hx -> Source/superhuman/server/hostsFileGenerator/StandaloneProvisionerHostsFileGenerator.hx
	modified:   Source/superhuman/server/provisioners/AbstractProvisioner.hx
	modified:   Source/superhuman/server/provisioners/AdditionalProvisioner.hx
	new file:   Source/superhuman/server/provisioners/CustomProvisioner.hx
	renamed:    Source/superhuman/server/provisioners/DemoTasks.hx -> Source/superhuman/server/provisioners/StandaloneProvisioner.hx
	modified:   Source/superhuman/theme/SuperHumanInstallerTheme.hx
	modified:   Templates/installer/nsis-installer.nsi

Unmerged paths:
  (use "git add/rm <file>..." as appropriate to mark resolution)
	both deleted:    Assets/provisioners/additional-prov/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml
	both deleted:    Assets/provisioners/additional-prov/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml
	both deleted:    Assets/provisioners/additional-prov/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml
	both deleted:    Assets/provisioners/additional-prov/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2
	both deleted:    Assets/provisioners/demo-tasks/0.1.20/scripts/ansible/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2
	both deleted:    Assets/provisioners/demo-tasks/0.1.22/scripts/provisioners/ansible/roles/hcl_domino_traveler_htmo/templates/htmo-traveler-access.json.j2
	both deleted:    Assets/provisioners/demo-tasks/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml
	both deleted:    Assets/provisioners/demo-tasks/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml
	both deleted:    Assets/provisioners/demo-tasks/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml
	both deleted:    Assets/provisioners/demo-tasks/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2
	both added:      Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml
	both added:      Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml
	both added:      Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml
	both added:      Assets/provisioners/hcl_domino_additional_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2
	both added:      Assets/provisioners/hcl_domino_standalone_provisioner/0.1.20/scripts/ansible/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2
	both added:      Assets/provisioners/hcl_domino_standalone_provisioner/0.1.22/scripts/provisioners/ansible/roles/hcl_domino_traveler_htmo/templates/htmo-traveler-access.json.j2
	both added:      Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_config/defaults/main.yml
	both added:      Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_genesis_applications/tasks/main.yml
	both added:      Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_jedi/tasks/main.yml
	both added:      Assets/provisioners/hcl_domino_standalone_provisioner/0.1.23/scripts/provisioners/ansible_collections/startcloud/hcl_roles/roles/domino_traveler_htmo/templates/htmo-traveler-access.json.j2
	both modified:   Source/SuperHumanInstaller.hx
	both modified:   Source/superhuman/managers/ProvisionerManager.hx
	both modified:   Source/superhuman/server/Server.hx
	both modified:   Source/superhuman/server/provisioners/ProvisionerType.hx
	both modified:   project.xml
```

![image](https://github.com/user-attachments/assets/2890c622-6e7d-4d09-bd8f-386f8c7d3c94)


Tagging #146 and #143
